scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -39,7 +39,8 @@
       <div class="card-header">
         <strong><a data-toggle="collapse" data-parent="#accordion" href="#collapseFilters">Filters</a></strong>
       </div>
-      <div class="card-body panel-collapse collapse {{ "show" if request.args.get("expand_search") }}" id="collapseFilters" >
+      <!--Expand filters form if filters were used in a previous POST request or the previous POST request form was not validated-->
+      <div class="card-body panel-collapse collapse {{ 'show' if request.args.get('expand_search') == "True" or expand_search== "True" }}" id="collapseFilters" >
```

Comment:
In this case there are 2 cases:
- request.args.get('expand_search') == "True"  # (when the filter form was submitted but not validated)
- expand_search== "True" # (when the filter form was submitted and validated and is returning filtered variants)

This is the only case having the expand_search passed also as args because the other pages don't have the filter validation in place yet (see #1314)